mungegithub/mungers/milestone-maintainer.go

```diff
@@ -330,19 +357,34 @@ func (m *MilestoneMaintainer) RegisterOptions(opts *options.Options) sets.String
 	return nil
 }
 
-func (m *MilestoneMaintainer) updateInterval() time.Duration {
-	if m.mode == milestoneModeSlush {
+func (m *MilestoneMaintainer) updateInterval(mode string) time.Duration {
+	if mode == milestoneModeSlush {
 		return m.slushUpdateInterval
 	}
-	if m.mode == milestoneModeFreeze {
+	if mode == milestoneModeFreeze {
 		return m.freezeUpdateInterval
 	}
 	return 0
 }
 
+func (m *MilestoneMaintainer) milestoneMode(obj *github.MungeObject) (string, string, bool) {
```

Comment:
Add a comment describing what the return values mean.

Reply:
Done